### PR TITLE
Doctests for user intent example notebook

### PR DESCRIPTION
**Pull request by ivarpruijn**
Reformats the feature-importance example notebook, and adds doctests.

### CODE REVIEW

File: notebooks/basic-user-intent.ipynb [hunk 1]
```diff
@@ -160,33 +154,50 @@
...
+    "- Users that spent most time on the site (the 90th percentile), and specifically in our documentation sections, are in the Implement phase.\n",
+    "- As there's a jump beyond the one minute mark at the 70th percentile, we assume that users in the 70th to 90th percentile duration in our documentation sections are in the Explore phase.\n",
+    "- The remaining users are Informing themselves about the product. Those users are spending less than 1:30 in the docs and/or spend any amount of time on our main website.\n",
+    "\n",
+    "Summarizing:\n",
     "\n",
     "| User intent | Root locations | Duration |\n",
     "| :--- | :--- | :--- |\n",
     "| 1 - Inform | *all sections other than the ones mentioned below* | *any time spent* |\n",
```
Comment (vard-obj) [inline notebooks/basic-user-intent.ipynb:198]: this line is redundant because the 3 other cases below are covering all the possibilities.

Comment (vard-obj) [inline notebooks/basic-user-intent.ipynb:198]: the first row in the table is redundant because the 3 other cases below are covering all the possibilities.

Comment (ivarpruijn) [reply notebooks/basic-user-intent.ipynb:198]: Isn't `all sections other than the ones mentioned below`  a different case than the other ones, which specifically look at `Docs: modeling, taxonomy, tracking, home`? Though I agree there's likely no matching data currently, as it should all fall within those root_locations if it's instrumented properly.

Comment (vard-obj) [reply notebooks/basic-user-intent.ipynb:198]: Right, let's leave it as it is.

**Review by vard-obj — APPROVED** [review]

**Pull request merged**